x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.test.tsx

```diff
@@ -31,4 +46,42 @@ describe('SearchExperience', () => {
     const wrapper = shallow(<SearchExperience />);
     expect(wrapper.find(SearchProvider).length).toBe(1);
   });
+
+  describe('customization modal', () => {
+    it('has a customization modal which can be opened and closed', () => {
+      const wrapper = shallow(<SearchExperience />);
+      expect(wrapper.find(CustomizationModal).exists()).toBe(false);
+
+      wrapper.find(CustomizationCallout).simulate('click');
+
+      expect(wrapper.find(CustomizationModal).exists()).toBe(true);
+
+      wrapper.find(CustomizationModal).prop('onClose')();
+
+      expect(wrapper.find(CustomizationModal).exists()).toBe(false);
```

Comment:
optional/minor nit, i'd consolidate your newline spacing here to associate actions w/ assertions

```suggestion
      wrapper.find(CustomizationCallout).simulate('click');
      expect(wrapper.find(CustomizationModal).exists()).toBe(true);

      wrapper.find(CustomizationModal).prop('onClose')();
      expect(wrapper.find(CustomizationModal).exists()).toBe(false);
```

Reply:
cb9774363365e60c2de3c4e706c0aaa5fc4cfc0e